compiler/rustc_incremental/src/persist/dirty_clean.rs

```diff
@@ -376,18 +377,18 @@ impl<'tcx> DirtyCleanVisitor<'tcx> {
                 continue;
             };
             self.checked_attrs.insert(attr.id);
-            for label in assertion.clean {
+            assertion.clean.to_sorted(&(), false).iter().for_each(|label| {
```

Comment:
and here?

afaict all uses of `to_sorted(&(), _)` should be replaceable with `to_sorted_stable_ord` 🤔 please go through all of these calls you're added and replace them